app/models/cx_collection.rb

```diff
@@ -4,7 +4,7 @@ class CxCollection < ApplicationRecord
   include AASM
 
   belongs_to :organization
-  belongs_to :service
+  belongs_to :service, counter_cache: true
```

Comment:
add counter_cache, so Service.cx_collections_count works and removes an additional query